packages/clay-css/src/scss/variables/_tables.scss

```diff
@@ -391,6 +391,7 @@ $c-tr-table-focus: () !default;
 $c-tr-table-focus: map-deep-merge(
 	(
 		td: (
+			outline: 'none',
```

Comment:
@matuzalemsteles This is a nitpick, but we have been using `outline: 0` to remove the outline.